src/spikeinterface/core/numpyextractors.py

```diff
@@ -127,7 +128,9 @@ def __init__(self, spikes, sampling_frequency, unit_ids):
         BaseSorting.__init__(self, sampling_frequency, unit_ids)
 
         self._is_dumpable = True
-        self._is_json_serializable = False
+        self._serializablility["json"] = False
+        # theorically this should be False but for simplicity make generators simples we still need this.
```

Comment:
Yeah, you guys were talking about this today. I am not sure I got it.

You don't wnat stuff to be pickable because they are heavy. You put all the spike_vector into disk. Is that correct?

Reply:
I wanted to avoid NumpySorting to also be pickle into file. But finally it is too hard!!
So lets keep `self._serializablility["pickle"] = True` for NumpySorting so the generated recording can be save into pickle which the most urgent need.